src/main/java/redis/clients/jedis/json/RedisJsonCommands.java

```diff
@@ -18,27 +18,27 @@ default String jsonSetLegacy(String key, Object pojo) {
     return jsonSet(key, Path.ROOT_PATH, pojo);
   }
 
-  default String jsonSet(String key, Object object, JsonSetParams params) {
-    return jsonSet(key, Path2.ROOT_PATH, object, params);
+  default String jsonSet(String key, Object json, JsonSetParams params) {
+    return jsonSet(key, Path2.ROOT_PATH, json, params);
   }
 
-  default String jsonSetWithEscape(String key, Object object, JsonSetParams params) {
-    return jsonSetWithEscape(key, Path2.ROOT_PATH, object, params);
+  default String jsonSetWithEscape(String key, Object json, JsonSetParams params) {
+    return jsonSetWithEscape(key, Path2.ROOT_PATH, json, params);
   }
 
   default String jsonSetLegacy(String key, Object pojo, JsonSetParams params) {
     return jsonSet(key, Path.ROOT_PATH, pojo, params);
   }
 
-  String jsonSet(String key, Path2 path, Object object);
+  String jsonSet(String key, Path2 path, Object json);
```

Comment:
If you prefer this param name here, make sure all similar methods have the same.